x-pack/plugin/security/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/remotecluster/RemoteCluster2SecuritySmokeIT.java

```diff
@@ -18,7 +21,7 @@
  * This test suite will be run twice: Once against the fulfilling cluster, then again against the querying cluster. The typical usage is to
  * conditionalize on whether the test is running against the fulfilling or the querying cluster.
  */
-public class RemoteClusterSecuritySmokeIT extends ESRestTestCase {
+public class RemoteCluster2SecuritySmokeIT extends ESRestTestCase {
```

Comment:
nit: we should probably be consistent with naming and I would prefer to avoid using "2" to mean the newer model.  I prefer (internally) referring to 1.0 model as legacy and no qualifiers for the new model.

Reply:
I renamed the other `RemoteClusterSecuritySmokeIT` to `LegacyRemoteClusterSecuritySmokeIT` and removed `2` from this class. I wanted to differentiate between the two classes by names because IntelliJ sometimes gets confused between them and causes issues in debugging. Thanks!